### ISSUE TITLE
Mobile Support for Volunteer Check In

### ISSUE THREAD

**Issue by davidhabib** [SalesforceFoundation/Volunteers-for-Salesforce#71]
How likely would you use a new feature that allows you to enter new contacts and track attendence at a volunteer event using a mobile phone or mobile tablet?

Very likely to use  42.45%  
Might use    37.74%  
Not interested in mobile solutions  13.21%  
Not Answered    6.60%

**Comment by mpusto** [comment]
This is now offered by a partner. Should we put in the backlog anyway? Seems lower priority given that customers who want it can use this. http://dftly.com/V4S_Mobile.html

**Comment by davidhabib** [comment]
yes, I think we should put this in the backlog.  the partners solution requires a custom mobile app, rather than being built into V4S using just Sites pages (ie, no custom app required).  The partner may also decide to start charging for their app.

**Comment by davidhabib** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x80000004DBKSAA4" target="_blank">W-016918</a>

**Comment by BarrieLPotter** [comment]
This would be extremely helpful!

**Comment by mbaizman** [comment]
@BarrieLPotter There is a partner solution for this: http://dftly.com/V4S_Mobile.html
